atom/browser/ui/cocoa/atom_menu_controller.mm

```diff
@@ -302,8 +307,7 @@ - (void)addItemToMenu:(NSMenu*)menu
 }
 
 // Called before the menu is to be displayed to update the state (enabled,
-// radio, etc) of each item in the menu. Also will update the title if
```

Comment:
comment removed as it's a remnant of dead code previously removed in https://github.com/electron/electron/pull/14939